src/components/Input/index.jsx

```diff
@@ -1,19 +1,15 @@
-import { MagnifyingGlass } from "@phosphor-icons/react";
+/* eslint-disable react/prop-types */
+
 import styles from "./styles.module.css";
 
-export function Input({ ...rest }) {
+export function Input({ borderNone, isNumber, ...rest }) {
   return (
-    <div className={styles["input-wrapper"]}>
-      <button className={styles["button-search"]}>
-        <MagnifyingGlass size={16} />
-      </button>
-
-      <input
-        {...rest}
-        type="text"
-        className={styles["input-header"]}
-        placeholder="Pesquisar produto"
-      />
-    </div>
+    <input
+      {...rest}
+      type={isNumber === "yes" ? "number" : "text"}
+      className={borderNone === "yes" ? `${styles["input-header_border-none"]}` : `${styles["input-header"]}`}
+      placeholder={isNumber === "yes" || isNumber === "not" ? "" : "Pesquisar produto"}
```

Comment:
I don't undestand this code, when this verification is true ? Plase, remove this  placeholder of the component and past on the page, we receive the ...rest, that is all the props of the input.

```
<Input 
    placeholder="pesquisar produto"
/>
```